Sunrise/ViewControllers/Products/StoreSelectionViewController.swift

```diff
@@ -182,4 +183,7 @@ extension StoreSelectionViewController: CLLocationManagerDelegate {
         viewModel?.userLocation.value = locations.last
     }
 
+    func locationManager(_ manager: CLLocationManager, didFailWithError error: Error) {
+        debugPrint(error)
```

Comment:
https://github.com/commercetools/commercetools-sunrise-ios/pull/162#issuecomment-313374862

>>> For my store and store selection screens, we use user's location, to show the distance from the available stores. Since we don't need to track every single movement / change, I've modified the code to track significant location changes, and save some battery.

The diff is not clean since there're a couple of other PRs open at this point; after you get a chance to have a glance, I'll proceed and merge them 👍